contentcuration/contentcuration/views/users.py

```diff
@@ -90,8 +90,8 @@ def send_invitation_email(request):
                     'channel_id': channel_id,
                     'invitation_key': invitation.id,
                     'channel': channel.name,
-                    'domain': request.META.get('HTTP_ORIGIN') or "https://{}".format(
-                        request.get_host() or Site.objects.get_current().domain),
+                    'domain': {"domain": "https://{}".format(
+                        Site.objects.get_current().domain)},
```

Comment:
I missed this, but looks like we copy and pasted the whole dictionary, not just the value for `"domain"`

Reply:
Also, do you have `pre-commit` installed? I would have expected it to reformat this code. I actually think it's missing from the Studio README/dev setup now that I look...

Reply:
There's also this outstanding issue: https://github.com/learningequality/studio/issues/2945 which means that we can't enforce the Python linting checks either.

Reply:
@rtibbles Ah right, I forgot we talked about handling that before the websockets work...

Reply:
We couldn't though, because my event log PR hadn't been merged! We'll do it once we merge the websockets work instead.